tests/drop_packets/drop_packets.py

```diff
@@ -142,40 +146,36 @@ def setup(duthost, tbinfo):
 
 
 @pytest.fixture
-def rif_port_down(duthost, setup, loganalyzer):
-    """ Disable RIF interface and return neighbor IP address attached to this interface """
+def rif_port_down(duthost, setup, fanouthosts):
+    """Shut RIF interface and return neighbor IP address attached to this interface.
+
+    The RIF member is shut from the fanout side so that the ARP entry remains in
+    place on the DUT."""
     wait_after_ports_up = 30
 
     if not setup["rif_members"]:
         pytest.skip("RIF interface is absent")
     rif_member_iface = setup["rif_members"].keys()[0]
 
-    try:
-        vm_name = setup["mg_facts"]["minigraph_neighbors"][rif_member_iface]["name"]
-    except KeyError as err:
-        pytest.fail("Didn't found RIF interface in 'minigraph_neighbors'. {}".format(str(err)))
+    vm_name = setup["mg_facts"]["minigraph_neighbors"][rif_member_iface].get("name", None)
+    pytest_assert(vm_name, 'Neighbor not found for RIF member "{}"'.format(rif_member_iface))
 
     ip_dst = None
     for item in setup["mg_facts"]["minigraph_bgp"]:
-        if item["name"] == vm_name:
-            if netaddr.valid_ipv4(item["addr"]):
-                ip_dst = item["addr"]
-                break
-    else:
-        pytest.fail("Unable to find neighbor in 'minigraph_bgp' list")
+        if item["name"] == vm_name and netaddr.valid_ipv4(item["addr"]):
+            ip_dst = item["addr"]
+            break
+    pytest_assert(ip_dst, 'Unable to find IP address for neighbor "{}"'.format(vm_name))
 
-    loganalyzer.expect_regex = [LOG_EXPECT_PORT_ADMIN_DOWN_RE.format(rif_member_iface)]
-    with loganalyzer as analyzer:
-        duthost.command("config interface shutdown {}".format(rif_member_iface))
+    fanout_neighbor, fanout_intf = fanout_switch_port_lookup(fanouthosts, rif_member_iface)
 
+    fanout_neighbor.shutdown(fanout_intf)
```

Comment:
you can still keep the loganalyzer check just changing the admin down to oper down msg. if you are not, remove those var. I think they aren't used by any other testcase

Reply:
That's a good point! I've added a check for oper status there now.